app/controllers/api/v1/ingredients_search_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::IngredientsSearchController < ApplicationController
+  respond_to :json
+
+  def index
+    respond_with IngredientFinder.fetch_ingredients(params[:search])
```

Comment:
@pwentz At quick glance this looks right. There's some interesting additional complexity here since we are touching a model and a service. Nothing jumps out as a red flag.